homeassistant/components/lastfm/const.py

```diff
@@ -13,3 +16,5 @@
 ATTR_TOP_PLAYED = "top_played"
 
 STATE_NOT_SCROBBLING = "Not Scrobbling"
+
+PLACEHOLDERS = {"api_account_url": "https://www.last.fm/api/account/create"}
```

Comment:
```suggestion
```
Put it where it belongs e.g. config flow

Reply:
I think I saw this at another integration aswell. In the future we can use this in the repairs. Or would you suggest moving it to config_flow.py now and moving it back to const when it is used in multiple files?

Reply:
Move it to `const` when there is use for it in multiple.